packages/shared/components/modals/FilterModal.svelte

```diff
@@ -1,8 +1,10 @@
 <script lang="ts">
     import { Modal, Text, FontWeight, Button, ButtonSize } from 'shared/components'
-    import type { Filter } from '@core/wallet'
     import { localize } from '@core/i18n'
-    import { activeProfileId } from '@core/profile'
+    import type { ActivityFilter, AssetFilter } from '@core/wallet/interfaces'
+    import { IProposalFilter } from '@contexts/governance'
+
+    type Filter = ActivityFilter | AssetFilter | IProposalFilter
```

Comment:
I would move this back into a separate file, since you're also using this in Filter.svelte